splice_spark/pom.xml

```diff
@@ -404,6 +404,82 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>dbaas</id>
+            <properties>
+                <scala.version>2.11.8</scala.version>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                      <artifactId>maven-jar-plugin</artifactId>
+                      <executions>
+                        <!-- Jar has to be shaded since the classifier is not recognized by shading -->
+                        <execution>
+                          <id>shaded-jar</id>
+                          <goals>
+                            <goal>jar</goal>
+                          </goals>
+                        </execution>
+                      </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                        <argLine>-Djava.library.path=${project.build.directory}/../../assembly/${envClassifier}/native/${os.detected.classifier}:/usr/local/lib/ -Dhbase.rootdir=${project.build.directory}/../../platform_it/target/hbase</argLine>
+                        </configuration>
+                    </plugin>
+                     <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-shade-plugin</artifactId>
+                        <version>3.2.1</version>
```

Comment:
nit: Remove `version` and define the plugin in `pluginManagement` in the parent/top-level pom.xml. Makes plugin management easier.